From 449b898e2116ac60003018ba93bea2ec56e34819 Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Mon, 8 May 2017 18:39:47 +0300
Subject: [PATCH 1760/5242] MLK-14798: arm: dts: imx6ul: Fix wm8960 codec
 master mode

commit  8860dbd990cca95dd8e3f6824cdb66db8cec4c3e from
https://source.codeaurora.org/external/imx/linux-imx.git

Commit 65e6b5f1b4a7 ("ASoC: wm8960: Fix playback in CPU DAI master
mode") broke wm8960 codec master mode by choosing "bad" SYSCLK values.

This patch partially reverts commit mentioned above by restoring the
SYSCLK values. It turns out that using params_physical_width instead of
params_width in the previous patch it is enough to fix CPU DAI mode.

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index 807c67d..6e23a86 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -147,7 +147,7 @@
 
 &clks {
 	assigned-clocks = <&clks IMX6UL_CLK_PLL4_AUDIO_DIV>;
-	assigned-clock-rates = <1179648000>;
+	assigned-clock-rates = <786432000>;
 };
 
 &cpu0 {
@@ -339,7 +339,7 @@
 	assigned-clocks = <&clks IMX6UL_CLK_SAI2_SEL>,
 			  <&clks IMX6UL_CLK_SAI2>;
 	assigned-clock-parents = <&clks IMX6UL_CLK_PLL4_AUDIO_DIV>;
-	assigned-clock-rates = <0>, <36864000>;
+	assigned-clock-rates = <0>, <12288000>;
 	fsl,sai-mclk-direction-output;
 	status = "okay";
 };
-- 
1.7.9.5

